pkg/config/config.go

```diff
@@ -127,20 +124,20 @@ func WithServer(host, port, resourceDir string, debug, verbose bool) Options {
 			ResourceDir: resourceDir,
 			Debug:       debug,
 			VerboseAPI:  verbose,
+			StartTime:   time.Now(),
 		}
 		return nil
 	}
 }
 
 // WithKafka returns an option that sets the kafka config
-func WithKafka(tls bool, version string, peers []string, topic string, channel chan message.Message) Options {
+func WithKafka(tls bool, version string, peers []string, topic string) Options {
```

Comment:
I notice in a couple other places, you renamed things to be non-kafka specific. May want to do it here too.

Reply:
Any suggestions?

Reply:
maybe WithStream? WithMessage?

Reply:
WithEventQueue, I think.

Reply:
This is bigger than you think... moving to an issue. https://github.com/sassoftware/event-provenance-registry/issues/71